src/lib.rs

```diff
@@ -628,27 +635,54 @@ pub use async_graphql_derive::Union;
 /// # Examples
 ///
 /// ```ignore
-/// use async_graphql::*;
+/// use async_graphql::prelude::*;
 ///
-/// #[Object]
+/// #[GqlObject]
 /// struct Event {
 ///     value: i32,
 /// }
 ///
 /// struct SubscriptionRoot;
 ///
-/// #[Subscription]
+/// #[GqlSubscription]
 /// impl SubscriptionRoot {
 ///     async fn value(&self, event: &Event, condition: i32) -> bool {
 ///         // Push when value is greater than condition
 ///         event.value > condition
 ///     }
 /// }
 /// ```
-pub use async_graphql_derive::Subscription;
+pub use async_graphql_derive::GqlSubscription;
 
 /// Define a DataSource
-pub use async_graphql_derive::DataSource;
+pub use async_graphql_derive::GqlDataSource;
 
 /// Define a Scalar
-pub use async_graphql_derive::Scalar;
+pub use async_graphql_derive::GqlScalar;
+
+mod deprecated {
+    pub use super::GqlContext as Context;
```

Comment:
Is there a way to annotate deprecations in rust?

Reply:
I'll check that tomorrow :)